test/test_site.rb

```diff
@@ -213,6 +213,7 @@ def generate(site)
       @site.process
       # exclude files in symlinked directories here and insert them in the
       # following step when not on Windows.
+      # rubocop:disable Lint/NestedPercentLiteral
```

Comment:
IMO this comment ought to be placed at the top level (just before `should "sort pages alphabetically" do`) and the enabler-comment at the corresponding end..

Reply:
Are there any best practice on this? Here I disable the minimum scope for the cops.

Reply:
Ideally we don't add any rubocop comments and lists files to exclude for now in the rubocop config file and anyone can pick a file and fix it like we did before.

Reply:
@Di r